desktop/linux/release-notes/index.md

```diff
@@ -10,6 +10,23 @@ This page contains information about the new features, improvements, known issue
 
 Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/projects/1){: target="_blank" rel="noopener" class="_"} to see what's coming next.
 
+## Docker Desktop 4.8.1
+2022-05-09
+
+> Download Docker Desktop
+>
+> [DEB](https://desktop-stage.docker.com/linux/main/amd64/78998/docker-desktop-4.8.0-amd64.deb) |
```

Comment:
Still needs a tiny correction: 4.8.0 -> 4.8.1 in the file names.

Reply:
Yes, this is going to be annoying, I think we should align Linux on the Windows and Mac practices, maybe?